src/vm/exceptionhandling.cpp

```diff
@@ -362,14 +362,26 @@ void ExceptionTracker::UpdateNonvolatileRegisters(CONTEXT *pContextRecord, REGDI
         pAbortContext = GetThread()->GetAbortContext();
     }
 
```

Comment:
Do you mean the case when the fAborting is true or the case when the UpdateNonvolatileRegisters is called with NULL context pointers?